src/client/sky.cpp

```diff
@@ -675,13 +675,13 @@ void Sky::draw_moon(video::IVideoDriver *driver, float moonsize, const video::SC
 		}
 	} else {
 		driver->setMaterial(m_materials[4]);
-		float d = (moonsize * 1.9) * m_moon_params.scale;
+		float d = (moonsize * 1.3) * m_moon_params.scale;
```

Comment:
The scaling seems rather arbitrary to me?

Reply:
Having a better looking, there was another subtle halo, so yes, this particular modification doesn't make sense. Reverted, thanks